Packs/DefaultPlaybook/Playbooks/Default_8_2.yml

```diff
@@ -253,6 +260,19 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+    scriptarguments:
+      Decode:
+        simple: "False"
```

Comment:
why not ? :)

Reply:
It's the default value and too advanced for Default flow, and we don't make use of decoded macros in it.